src/docs/deployment.md

```diff
@@ -191,6 +191,15 @@ Includes persisted cache across builds. Using [`peaceiris/actions-gh-pages`](htt
 <ol>
 <li>Go to your repository’s Settings on GitHub.</li>
 <li>In the GitHub Pages section change:<ul><li>Source: <code>Deploy from a branch</code></li><li>Branch: <code>gh-pages/(root)</code></li></ul></li>
+<li>Add "build-ghpages" command to your <details><summary><code>package.json</code> scripts section</summary>
+
+```json
+"scripts": {
+  "build-ghpages": "npx @11ty/eleventy --pathprefix=/your-repo-name/",
```

Comment:
Just a suggestion.

Since the repo name is known from the github actions, I would add `--pathprefix=${{ github.event.repository.name }}` directly there. Changing line 235 to:

```yml
      - run: npm run build --pathprefix=${{ github.event.repository.name }}
```

Then we would just need to add a note "please make sure the build script is set in your `package.json`.